File: 2545-sort-the-students-by-their-kth-score/2545-sort-the-students-by-their-kth-score.cpp
class Solution {
public:
    vector<vector<int>> sortTheStudents(vector<vector<int>>& sc, int k) {
        vector<pair<int,int>>v;
        for(int i=0;i<sc.size();++i){
            v.push_back({sc[i][k],i});
        }
        sort(begin(v),end(v));
        // reverse(begin(v),end(v));
        vector<vector<int>>ans;
        for(int i=sc.size()-1;i>=0;--i){
            ans.push_back(sc[v[i].second]);
        }
        return ans;
    }
};

File: 2545-sort-the-students-by-their-kth-score/2545-sort-the-students-by-their-kth-score.py
class Solution:
    def sortTheStudents(self, scr: List[List[int]], k: int) -> List[List[int]]:
        temp =[]
        for i in range(len(scr)):
            temp.append([scr[i][k],i])
            # print(scr[i][k])
        temp.sort(reverse=True)
        # print(temp)
        res = []
        for i in range(len(temp)):
            res.append(scr[temp[i][1]])
        return res